#request
uint8 payload_index
uint8 exposure_mode
uint8 exposure_compensation
uint16 aperture
uint8 iso_data
uint8 shutter_speed
float32 x #range from 0~1, top left corner is origin point
float32 y #range from 0~1, top left corner is origin point
---
#response
bool result